src/routes/(auth)/inscription/+page.svelte

```diff
@@ -47,6 +50,7 @@
 
 		<Button>Créer un compte</Button>
 
+		<Link href="/">Retour</Link>
```

Comment:
It's doesnt look like the figma.

<img width="510" alt="Capture d’écran 2023-02-25 à 15 29 32" src="https://user-images.githubusercontent.com/46904617/221362378-c8a8bf3e-329a-4ea9-9178-3ed102013320.png">